components/molecule/rating/src/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import Star from './components/Star'
+
+const SIZES = {
+  SMALL: 'small',
+  MEDIUM: 'medium',
+  LARGE: 'large'
+}
+
+const BASE_CLASS = `sui-MoleculeRating`
+const CLASS_CONTAINER_STARS = `${BASE_CLASS}-containerStars`
+const CLASS_LABEL = `${BASE_CLASS}-label`
+const CLASS_LINK = `${BASE_CLASS}--withLink`
+
+const MoleculeRating = ({
+  numStars = 5,
+  value,
+  size = SIZES.SMALL,
+  label,
+  href,
+  target,
+  link,
+  linkFactory: Link = ({children, ...rest} = {}) => <a {...rest}>{children}</a>
+}) => {
+  const className = cx(BASE_CLASS, `${BASE_CLASS}--${size}`, {
+    [CLASS_LINK]: link
+  })
+
+  const labelLink = link ? (
+    <Link
```

Comment:
Not sure about that but, you are defining a linkFactory that is a component named Link, but if someone wants to pass a new linkFactory, has it to be named Link as well? If so, it should be explained in the API documentation.

Reply:
What you can pass is an anonymous component as a prop that is "aliased" as `Link` inside the code for a better understanding